README.md

```diff
@@ -1,57 +1,84 @@
 # Create Block Theme
-A WordPress plugin to create block themes.
+
+Welcome to Create Block Theme - a WordPress plugin to create block themes from within the Site Editor.
+
+It works alongside features that are already available in the Site Editor to enhance the workflow for creating block themes. After being tested in this plugin, some of the features included here may be moved into the Site Editor itself.
 
 This plugin allows you to:
-- Create a new theme, blank theme, child theme or style variation.
-- Embed Google Fonts in your theme
-- Embed local font assets in your theme
 
-The plugin is development only — not intended for use on production websites, but used as a tool to create new themes.
+-   Create a new theme, blank theme, child theme, or style variation from within the Site Editor
+-   Embed Google Fonts locally in your theme
+-   Embed local font assets in your theme
+
+Learn more about [how to use the plugin](#how-to-use-the-plugin) or [how to contribute](#how-to-contribute).
+
+_The plugin is for development only — it is not intended for use on production websites, but to be used as a tool to create new themes._
```

Comment:
While we're here, what do you think we should move this line higher, perhaps below line 5?

Reply:
Yep, sounds good! Done in https://github.com/WordPress/create-block-theme/pull/220/commits/875db3dee1da0573fcac3c78575e1aebe0714c08.